src/mbed_tools/project/project.py

```diff
@@ -55,11 +59,11 @@ def deploy_project(path: pathlib.Path, force: bool = False) -> None:
         force: Force overwrite uncommitted changes. If False, the deploy will fail if there are uncommitted local
```

Comment:
Pre-existing issue: There is no `project_revision` parameter, so it's a bit odd to attempt documenting it here. Should fix in separate commit.

Reply:
This was fixed in ff5b5a265a972e2906655dbe6ef7ee9c9902f4ab. I need to rebase.

Reply:
Rebased.

Reply:
I still see `project_revision: Revision of the Mbed project to check out.` in `project.py`.

Reply:
Oh, sorry it wasn't fixed upstream. I've fixed it in 1dd5e7b989339817cacaaf1ab4a0757cec35cb23